app/models/role.rb

```diff
@@ -2,4 +2,6 @@
 
 class Role < ApplicationRecord
   validates :name, presence: true
+
+  has_many :users
 end
```

Comment:
I would add an seed with default roles - like `Admin` and `Client` - especially as those as required to create user